### ISSUE TITLE
Exception on Linux Mint (Ubuntu 20)

### ISSUE THREAD

**Issue by baumschubser** [melianmiko/OpenFreebuds#15]
On my Linux Mint 20.3 (based on Ubuntu 20.03) system, I install the .deb package (openfreebuds_0.12.1-1_all.deb) and run `openfreebuds`.
Program start fails with this error:
```
Traceback (most recent call last):
  File "/usr/bin/openfreebuds", line 5, in <module>
    import ofb_launcher
  File "/opt/openfreebuds/ofb_launcher.py", line 11, in <module>
    from openfreebuds import event_bus, manager, cli_io
  File "/opt/openfreebuds/openfreebuds/manager.py", line 6, in <module>
    import openfreebuds.device
  File "/opt/openfreebuds/openfreebuds/device/__init__.py", line 2, in <module>
    from openfreebuds.device import huawei
  File "/opt/openfreebuds/openfreebuds/device/huawei/__init__.py", line 1, in <module>
    from openfreebuds.device.huawei.profiles import FreeLaceProDevice, FreeBuds4iDevice
  File "/opt/openfreebuds/openfreebuds/device/huawei/profiles.py", line 1, in <module>
    from openfreebuds.device.huawei.generic.spp_device import GenericHuaweiSppDevice
  File "/opt/openfreebuds/openfreebuds/device/huawei/generic/spp_device.py", line 8, in <module>
    from openfreebuds.device.huawei.generic.spp_handler import HuaweiSppHandler
  File "/opt/openfreebuds/openfreebuds/device/huawei/generic/spp_handler.py", line 1, in <module>
    from openfreebuds.device.huawei.generic.spp_package import HuaweiSppPackage
  File "/opt/openfreebuds/openfreebuds/device/huawei/generic/spp_package.py", line 4, in <module>
    class HuaweiSppPackage:
  File "/opt/openfreebuds/openfreebuds/device/huawei/generic/spp_package.py", line 5, in HuaweiSppPackage
    def __init__(self, cmd: bytes, parameters: list[tuple[int, bytes | int]] = None):
TypeError: unsupported operand type(s) for |: 'type' and 'type'

```

**Comment by melianmiko** [comment]
Which Python version do you have? The latest versions of OpenFreebuds require Python 3.10

**Comment by baumschubser** [comment]
It's 3.8, the default on Ubuntu 20 and its derivatives. I tried to install Python 3.10 via PPA or pyenv, but I did not succeed (I guess I did not try hard enough).

Maybe add Python 3.10 to the dependencies? Not sure how well this would play with pyenv.

Anyways, I'll postpone the use until I update my system. Thanks for the software anyways! The crappy 500+ MB "AI Life" app that I need just to change ANC settings is really the worst part of the headphones.
